### DISCUSSION TITLE
Mathjax Implementation

### DISCUSSION THREAD

**Discussion by underthelights** [retypeapp/retype#139] [Q&A] [answered]
Hello, I tried to use a JavaScript math library 'MathJax' ​to express some mathematical notations on 'retype'. But I cannot see any changes after I applied some Github repo folder settings, and cannot find any comments from retype.com. It's probably because I'm not good enough, but is there a way?

**Comment by fabriciomurta** [comment, marked as answer]
We do have this under our radar, take a look at #73; there are also some referenced discussions about suport on mathjax. You may want to add something to the issue or simply +1 it.

So, to answer your question, when support is implemented, you wouldn't need to be "good enough" it should be easily usable without much effort! But for now, it's simply not going to work nicely -- if at al.

**Reply by underthelights** [reply]
Yeah, thanks for your effort. Retype is one of the best platform to design my own blog prototype. Thank You!

**Reply by geoffreymcgill** [reply]
We will implement MathJax support soon. It's a super high priority for us.

Retype can now also be used as a blogging platform. We will also be publishing more details about blogging with Retype very soon, hopefully later this week. Just add your `.md` pages into a `/blog/` folder, something like `/blog/2021-08-15-welcome-to-retype.md`.

If you have a blog, you might also want to add a `Blog` link into the top [links](https://retype.com/configuration/project/#links) with the following config in your `retype.yml` project configuration file:

```yml
links:
  - text: Blog
    link: /blog/
```

Hope this helps.

**Reply by geoffreymcgill** [reply]
KaTex math formula support has been added to Retype and will be available within the upcoming `v1.10.0` release. 

Hope this helps.
